tools/oca_projects.py

```diff
@@ -10,6 +10,8 @@
 
 """
 
+from github_login import login
```

Comment:
I understand the following that the script should still work if there's no github login, then it should fall back to the hardcoded list. The same should apply in my opinion if the module is not available